bower.json

```diff
@@ -9,12 +9,9 @@
     "ember-qunit": "0.4.9",
     "ember-qunit-notifications": "0.0.7",
     "ember-resolver": "~0.1.20",
-    "jquery": "^1.11.3",
+    "jquery": "<=2.1.4",
     "loader.js": "ember-cli/loader.js#3.2.1",
-    "materialize": "~0.97.0",
+    "materialize": "<=0.97.5",
```

Comment:
This for contributors. Whoever will clone this repo and runs the dummy app, won't see any icons.